pkg/cmd/cli/cmd/startbuild_test.go

```diff
@@ -60,6 +61,7 @@ func TestStartBuildWebHook(t *testing.T) {
 		Out:          buf,
 		ClientConfig: cfg,
 		FromWebhook:  server.URL + "/webhook",
+		Mapper:       registered.RESTMapper(),
```

Comment:
Before this PR, mapper was not required. It is now with the introduction of:
```kcmdutil.PrintSuccess(o.Mapper, o.ShortOutput, o.Out, "build", newBuild.Name, "started")```

If, by "before", you mean interim PR commits, it is because startbuild_test.go was not working but that failure was being masked by the other flakes.